src/components/whiteboards/WhiteboardCreate.vue

```diff
@@ -1,12 +1,215 @@
+          <div class="help-block" data-ng-messages="whiteboardsCreateForm.whiteboardsCreateTitle.$error">
+            <div data-ng-message="required">Please enter a title</div>
+            <div data-ng-message="maxlength">A title can only be 255 characters long</div>
+          </div>
+        </div>
+      </div>
+      <div class="form-group">
+        <label for="whiteboards-create-collaborators" class="col-sm-2 control-label">Collaborators</label>
+        <div class="col-sm-10">
+          <oi-select
+            class="cleanMode"
+            oi-options="user.id as user.canvas_full_name for user in users track by user.id | orderBy: 'canvas_full_name.toLowerCase()'"
+            oi-select-options="{'searchFilter': 'usersSearch', 'dropdownFilter': 'usersDropdown'}"
+            ng-model="whiteboard.members" multiple placeholder="Enter the names of the people you want to add to this whiteboard" tabindex="-1">
+          </oi-select>
+        </div>
+      </div>
+      <div class="form-group text-right">
+        <div class="col-sm-offset-2 col-sm-10 btn-row-reversed">
+          <button type="submit" class="btn btn-primary">Create whiteboard</button>
+          <a data-ng-href="/whiteboards" class="btn btn-default">Cancel</a>
+        </div>
+      </div>
+    </form>
+    -->
   </div>
 </template>
 
 <script>
+import BackToWhiteboards from '@/components/util/BackToWhiteboards'
+import Context from '@/mixins/Context'
+import PageTitle from '@/components/util/PageTitle'
+import Utils from '@/mixins/Utils'
+import {createWhiteboard} from '@/api/whiteboards'
+
 export default {
-  name: 'WhiteboardCreate'
+  name: 'WhiteboardCreate',
+  components: {BackToWhiteboards, PageTitle},
+  mixins: [Context, Utils],
+  computed: {
+    disableSave() {
+      return !this.$_.trim(this.title || '')
+    }
+  },
+  data: () => ({
+    collaborators: ['Sandra Adams', 'Britta Holt'],
+    isSaving: false,
+    isUpdating: false,
+    people: [
+      {header: 'Group 1'},
+      {name: 'Sandra Adams', group: 'Group 1', avatar: 'https://cdn.vuetifyjs.com/images/lists/1.jpg'},
+      {name: 'Ali Connors', group: 'Group 1', avatar: 'https://cdn.vuetifyjs.com/images/lists/2.jpg'},
+      {name: 'Trevor Hansen', group: 'Group 1', avatar: 'https://cdn.vuetifyjs.com/images/lists/3.jpg'},
+      {name: 'Tucker Smith', group: 'Group 1', avatar: 'https://cdn.vuetifyjs.com/images/lists/2.jpg'},
+      {divider: true},
+      {header: 'Group 2'},
+      {name: 'Britta Holt', group: 'Group 2', avatar: 'https://cdn.vuetifyjs.com/images/lists/4.jpg'},
+      {name: 'Jane Smith ', group: 'Group 2', avatar: 'https://cdn.vuetifyjs.com/images/lists/5.jpg'},
+      {name: 'John Smith', group: 'Group 2', avatar: 'https://cdn.vuetifyjs.com/images/lists/1.jpg'},
+      {name: 'Sandra Williams', group: 'Group 2', avatar: 'https://cdn.vuetifyjs.com/images/lists/3.jpg'},
+    ],
```

Comment:
autocomplete mock data taken from https://vuetifyjs.com/en/components/autocompletes/#item-and-selection